pkgs/servers/home-assistant/custom-components/luxer_one/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  ruff,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "tjhorner";
+  domain = "luxer";
+  version = "0-unstable-2023-05-06";
```

```suggestion
  version = "0-unstable-2023-03-27";
```

This is the date of the last commit.

https://github.com/tjhorner/home-assistant-luxer-one/commit/f6a810034ab76e6a8635de755c4a1750e86b1674